hypertrace-core-graphql-attribute-scope/build.gradle.kts

```diff
@@ -2,10 +2,6 @@ plugins {
   `java-library`
 }
 
-java {
-  sourceCompatibility = JavaVersion.VERSION_11
```

Comment:
Sorry about the noise here. This is configured at the top level across all projects already, so these are just dupes:
https://github.com/hypertrace/hypertrace-core-graphql/blob/c06ef820b3b430680fa444918af9e029a0c91144/build.gradle.kts#L12-L17